client/src/components/Mention/MoodIcon/MoodIcon.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import MoodIconSmile from "@material-ui/icons/SentimentSatisfiedOutlined";
+import SentimentDissatisfiedIcon from "@material-ui/icons/SentimentDissatisfied";
+import SentimentVeryDissatisfiedIcon from "@material-ui/icons/SentimentVeryDissatisfied";
+
+const useStyles = makeStyles({
+  icon: {
+    position: "absolute",
+    right: 16,
+    top: 16,
+    fontSize: "1.6rem",
+    color: "royalblue",
+  },
+});
+
+export default function MoodIcon({ mood }) {
+  const classes = useStyles();
+
+  switch (mood) {
+    case "good":
+      return <MoodIconSmile className={classes.icon} />;
+    case "average":
+      return <SentimentDissatisfiedIcon className={classes.icon} />;
+    case "bad":
+      return <SentimentVeryDissatisfiedIcon className={classes.icon} />;
+    default:
+      return null;
```

Comment:
Will the styling of the Mention be messed up if we return null here? Might be good to just pick a case to be the default.

Reply:
I don't think it will mess with the layout because I set the mood icon to be absolutely positioned (and therefore outside of the normal flow), but I didn't test it. I'll test it and respond here.
However, if you'd prefer to have a default so that every mention has an icon, maybe I can set the default to be the "average" symbol?

Reply:
It doesn't mess with the flow. But I'll add a default so that they all look homogeneous.